### PR TITLE
BZ1854206: moving telemetry module in installation assemblies

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1854206

4.6+

Sample preview: https://deploy-preview-35135--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-default.html#cluster-entitlements_installing-aws-default

### CODE REVIEW

**Comment by juzhao** [comment]
@kalexand-rh the bug is reported for 4.4, why we don't want to merge the fix to 4.4 and 4.5

**Comment by kalexand-rh** [comment]
@juzhao, 4.4 and 4.5 are no longer active versions. We don't update them except in very special circumstances. (I have not seen an update to an EOL version.)

File: installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc [hunk 1]
```diff
@@ -57,6 +57,10 @@ include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
 
 include::modules/olm-restricted-networks-configuring-operatorhub.adoc[leveloffset=+1]
 
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+* See xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service.
```
Comment (jeana-redhat) [inline installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc:62]: This one doesn't have an add'l resources header

File: installing/installing_gcp/installing-restricted-networks-gcp.adoc [hunk 2]
```diff
@@ -104,6 +106,10 @@ include::modules/installation-gcp-user-infra-adding-ingress.adoc[leveloffset=+1]
 
 include::modules/installation-gcp-user-infra-completing.adoc[leveloffset=+1]
 
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+* See xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service.
```
Comment (jeana-redhat) [inline installing/installing_gcp/installing-restricted-networks-gcp.adoc:111]: This one doesn't have an add'l resources header

File: installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc [hunk 3]
```diff
@@ -95,6 +97,10 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
 
 include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+* See xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service.
```
Comment (jeana-redhat) [inline installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc:102]: This one doesn't have an add'l resources header

**Review by jeana-redhat — COMMENTED** [review]
There sure are a lot of Install topics :sweat_smile: Just found a couple missing headers for the add'l resources link.

**Pull request merged**